src/ZulipMobile.js

```diff
@@ -16,14 +13,6 @@ export default class ZulipMobile extends Component {
 
   componentWillMount() {
     restore(() => {
-      store.subscribe(() => {
```

Comment:
We can't get rid of this. The idea here is that when switching accounts we need to retrigger an initial fetch. The `SWITCH_ACCOUNT` action causes, among other things, `state.app.needsInitialFetch` to be set to true, and that immediately trigger this reload.